lib/Listener/SabrePluginAuthInitListener.php

```diff
@@ -43,6 +43,11 @@ public function handle(Event $event): void {
 		}
 
 		$server = $event->getServer();
+
+		if (!str_starts_with($server->getRequestUri(), 'photospublic/')) {
+			return;
+		}
+
```

Comment:
I could not reproduce as my desktop client do not show the main dialog to add an account right now :thinking: 
@CarlSchwan could you confirm that this fixes the issue ?